Zend/tests/bug72496.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Bug #72496 (declare public method with signature incompatible with parent private method should not throw a warning)
+--FILE--
+<?php
+class Foo
+{
+    private function getName()
+    {
+        return 'John';
+    }
+}
+
+class Bar extends Foo
+{
+    public function getName($extraArgument)
```

Comment:
The problem also arises when the function is `protected`, can you add a test for that ?